These are TWAS data mining results from CF GWAS summary results using FUSION as outlined from Gusev lab: http://gusevlab.org/projects/fusion/. Briefly, updated GWAS summary statistics (https://github.com/danghunccf/CF-GWAS-dataMiningPaper/tree/master/GWAS_results), were formatted as input for FUSION => CF lung disease severity (KNoRMA) association testing with FUSION based on prediction models derived from 48 GTEx human tissues (v7 release) => Results from multiple tissues were combined through 2 meta-analysis methods of multiple p-values: HarmonicMeanPvalue and OmnibusPvalue (TWAS) => Maximal p-value and mean zscore (as effect size) between the 2 methods were compiled and used for comparison with PrediXcan and GWAS.
NOTE: Some differences compared to PrediXcan results reported - the TWAS models used only cis-SNP 0.5 Mbase flanking a gene to model genetically regulated gene expression; and we used summary GWAS statistics derived from the entire CF GWAS cohorts.

* CF lung disease severity association test results from individual tissues using FUSION were split into chromosomes and stored in the .tar.gz file.
* Gene annotation and compiled multiple tissue p-values and zscores were saved as text files.
* Meta-analysis results combining multiple tissue p-values with either HarmonicMeanPvalue or OmnibusPvalue, and the maximal p-value between the 2 were stored as text files (keywords: comboHMP, omnibusP, and maxP in filenames).
* Pre-ranked GSEA pathway enrichment analysis was performed by max p-value rank using Bioconductor fgsea with 1 million permutations (keyword: fgsea1m in filenames).
